Tests/SwiftLintFrameworkTests/ModifierOrderTests.swift

```diff
@@ -144,4 +144,21 @@ class ModifierOrderTests: XCTestCase {
         verifyRule(descriptionOverride,
                    ruleConfiguration: ["preferred_modifier_order": ["override", "acl", "final"]])
     }
+    #if swift(>=4.1.0)
```

Comment:
Should we do a runtime check (using `SwiftVersion.current`) instead of at compile time?